highlight/geany/geany-README.md

```diff
@@ -13,7 +13,7 @@ where "user" is your username.
 
```

Comment:
I think there's some text above this that should be removed (committer name/timestamps/dco)